README.md

```diff
@@ -240,6 +240,48 @@ const render = () => html`
 `;
 ```
 
+#### `asyncAppend(asyncIterable)` and `asyncReplace(asyncIterable)`
+
```

Comment:
I think maybe a bit of text at the beginning of this readme section could help the readers mind flow into what is about to be discussed:

"Asynchronous iterators offer the ability to continuously offer new data to be rendered into an element. lit-html offers two ways to consume that data into your element:"

Reply:
added something similar